PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-lib/src/main/resources/com/microsoft/intellij/ui/messages/messages.properties

```diff
@@ -1481,9 +1484,11 @@ webapp.deploy.validate.noResourceGroup=Resource Group not provided.
 webapp.deploy.validate.noAppServicePlan=App Service Plan not provided.
 webapp.deploy.validate.noLocation=Location not provided.
 webapp.deploy.validate.noPricingTier=Pricing Tier not provided.
+webapp.deploy.validate.loading=Loading meta-data of target web app
```

Comment:
`webapp.validation_runconfiguration.loading`, e.g. meta-data of target webapp is still loading...`